task-sdk/tests/task_sdk/execution_time/test_task_runner.py

```diff
@@ -2538,6 +2564,118 @@ def execute(self, context):
         assert listener.state == [TaskInstanceState.RUNNING, TaskInstanceState.FAILED]
         assert listener.error == error
 
+    def test_listener_access_outlet_event_on_running_and_success(self, mocked_parse, mock_supervisor_comms):
+        """Test listener can access outlet events through invoking get_template_context() while task running and success"""
+        listener = self.CustomOutletEventsListener()
+        get_listener_manager().add_listener(listener)
+
+        test_asset = Asset("test-asset")
+        test_key = AssetUniqueKey(name="test-asset", uri="test-asset")
+        test_extra = {"name1": "value1", "nested_obj": {"name2": "value2"}}
+
+        class Producer(BaseOperator):
+            def execute(self, context):
+                outlet_events = context["outlet_events"]
+                outlet_events[test_asset].extra = {"name1": "value1", "nested_obj": {"name2": "value2"}}
```

Comment:
```suggestion
                outlet_events[test_asset].extra = test_extra
```

maybe just reuse the `test_extra` variable?

Reply:
Yeah. The hard-coded dictionary looks redundant here. Will update it.